packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js

```diff
@@ -46,7 +29,7 @@ const MagicLinkWrapper = ({ children, target }) => {
       title={target}
       titleEllipsis
       subtitle={children}
-      icon={<Envelope />}
+      icon={<span style={{ fontSize: 32 }}>✉️</span>}
```

Comment:
wondering, why don't we use the DS icon? 👀
```suggestion
      icon={Envelop}
```

Reply:
It's not the same icon! So wasn't sure whether to do that or not? 👀

Reply:
ah right my bad, well still feel it's a shame we don't use DS icons but that could be another discussion!

Reply:
Something to add to our backlog?

Reply:
yup 🚀